easybuild/easyconfigs/q/QD/QD-0.8.9_remove_pybind_requirement.patch

```diff
@@ -0,0 +1,13 @@
+Requirement on pybind11 is unncessarily strict.
+Author: micketeer@gmail.com
+--- setup.py.orig	2021-08-31 11:25:48.110514794 +0200
++++ setup.py	2021-08-31 11:25:53.937501309 +0200
+@@ -325,7 +325,7 @@
+           ext_package='qd.cae',  # where to place c extensions
+           ext_modules=[dyna_extension],
+           install_requires=['numpy>=1.8',
+-                            'diversipy', 'pybind11==2.1.1', 'h5py'],

```

Comment:
@Micket Wouldn't it be easier w.r.t. maintaining this to avoid the use of a patch for this?

Instead:

```python
    (name, version, {
        'source_tmpl': '%(namelower)s-%(version)s.tar.gz',
        'checksums': ['711fcc0772011dccddbd971407d79b9bea06f11745162066c6dbbca0f6ff5211'],
        # requirement on pybind11 is unncessarily strict
        'preinstallopts': "sed -i 's/pybind11==/pybind11>=/g' setup.py && ",
    }),

Reply:
@Micket any thoughts on this?

Reply:
Well, they haven't made new releases in years, so either way I don't think it's going to be a maintenance burden. I'll change if it you guys think it's better with sed.

Reply:
Well, it would be better to avoid `sed` and have proper runtime patching support in EasyBuild itself, but that's another matter... :)

Since there haven't been any recent releases, the patch is good enough I guess, we can adjust the approach later if needed.